packages/@lwc/ssr-runtime/src/render-text-content.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+/**
+ * Given an object, render it for use as a text content node.
+ * @param value
+ */
+export function renderTextContent(value: any): string {
```

Comment:
```suggestion
export function massageTextContent(value: any): string {
```

Reply:
```suggestion
export function normalizeTextContent(value: unknown): string {
```
The function doesn't do the rendering itself, it just converts the value to a standard format.